LRCI-6455 Build cache testing

kenjiheigel opened the pull request.

yuriymr committed:
LPD-66671 Removing Velocity Template from Workflow

kenjiheigel committed:
LRCI-6455 Prioritize build object build reports if available

kenjiheigel committed:
LRCI-6455 Optimize further for playwright test results

kenjiheigel committed:
LRCI-6455 Always check the build object for build reports first, then the cache

kenjiheigel committed:
LRCI-6455 list.contains() can be expensive performance wise; use hashset instead to ensure unique objects

kenjiheigel committed:
LRCI-6455 We need to store multiple DownstreamBuildReports per axis name for this to function correctly

kenjiheigel committed:
Revert "LRCI-6455 Optimize further for playwright test results"

This reverts commit 553eef49873de0a8c56615bb313c98d7c8c68564.

kenjiheigel committed:
Revert "LRCI-6455 Always check the build object for build reports first, then the cache"

This reverts commit 3261b5244556da369b2e4d56a2360667a9bccf36.

kenjiheigel committed:
Revert "LRCI-6455 Prioritize build object build reports if available"

This reverts commit 6f6df2f6fa862ff202eaea86e0a94a2d4775cd02.

kenjiheigel committed:
LRCI-6455 Thread safety

marianoalvarosaiz committed:
LPD-67523 Create test to ensure that the invocation to assetHelper filters the collection subtypes.

marianoalvarosaiz committed:
LPD-67523 Take into account collection subtypes, if exist, to filter.

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>stable</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-42.liferay.com/job/test-portal-acceptance-pullrequest(master)/881/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-42.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/881/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-42.liferay.com/job/test-portal-acceptance-pullrequest(master)/880">test-portal-acceptance-pullrequest(master)#880</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-42.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/880/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>stable<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/kenjiheigel/liferay-portal/pull/1960">kenjiheigel#1960</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/333996776">[master] ci:test:stable - kenjiheigel &gt; kenjiheigel - PR#1960 - 2025-10-06[11:10:05]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>333996776</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-42.liferay.com/job/test-portal-acceptance-pullrequest(master)/880/">test-portal-acceptance-pullrequest(master)#880</a><br/></div></div>

marianoalvarosaiz committed:
LPD-67736 Modify UpgradeAssetDisplayPageEntry to make use of database indexes. No functional change.

marianoalvarosaiz committed:
LPD-67736 Modify BaseUpgradeAssetDisplayPageEntry to make use of database indexes. No functional change.

marianoalvarosaiz committed:
LPD-67736 Use proper default value.

marianoalvarosaiz committed:
LPD-67736 Create temporal index, since there are three deletions that will use it.

marianoalvarosaiz committed:
LPD-67736 SF, reuse.

marianoalvarosaiz committed:
LPD-67736 processConcurrently BaseUpgradeAssetDisplayPageEntry.

igor-franca committed:
LPD-67878 Changing the test for the new expected behavior related to LPD-66020

igor-franca committed:
LPD-66421 Remove getByTestId from object-web

gianmarcobrunialti committed:
LPD-68466 Restored guest checkout on new account selector

gianmarcobrunialti committed:
LPD-68466 Reindex order on reconciliation

marianoalvarosaiz committed:
LPD-67736 processConcurrently UpgradeAssetDisplayPageEntry.

DavysonMelo committed:
LPD-65888 Fix test errors

marianoalvarosaiz committed:
LPD-67736 Semantic Versioning

daiazius committed:
LPD-67917 Update build.gradle to add directive to preserve whitespaces

daiazius committed:
LPD-67917 Regen sanitizer configurations

daiazius committed:
LPD-67917 Add test

thiagobuarqque committed:
LPD-67929 rename DYNAMIC to BATCH

thiagobuarqque committed:
LPD-67929 use batch as the default type

thiagobuarqque committed:
LPD-67929 remove everything directly related to STATIC segments

anabxalves committed:
LRSD-11375 - Adjusting Announcements Template and creating a Table of Contents reusable fragment

thiagobuarqque committed:
LPD-67929 add REAL_TIME segment type

daiazius committed:
LPD-57780 Prevent users from saving the configuration if a field required for SimpleCaptcha to work is empty

daiazius committed:
LPD-57780 buildLang

thektan committed:
LPD-66625 Add aria-label to select all checkbox

sandrodw3 committed:
LPD-68122 Move logic to function

sandrodw3 committed:
LPD-68122 Add language keys

sandrodw3 committed:
LPD-68122 buildLang

adolfopa committed:
LPD-68210 Set default permissions for space and regular roles

adolfopa committed:
LPD-68210 SF

adolfopa committed:
LPD-68210 Remove duplication

adolfopa committed:
LPD-68210 Get object definition inside private method

sandrodw3 committed:
LPD-68122 Unmap input fragments when moving them to new parent

brbalazs committed:
LPD-67582 Added new menuitem in BaseSectionDisplayContext

brbalazs committed:
LPD-67582 Implemented reset permission menuitem logic on UI

brbalazs committed:
LPD-67582 Updated Language.properties

daiazius committed:
LPD-57780 Update test to add new case

abfranca committed:
LRSD-11226 Fix error when trying to upload a new file after a failure

abfranca committed:
LRSD-11378 Fix multiple comments added to ticket after concurrent upload of same file

JhossephG committed:
LPD-68036 feat: create validation for SEO box type

thektan committed:
LPD-66625 Match labels to the checkbox function

Clicking on the checkbox when there are partially selected items clear the items so the label should be "Clear Selection"

dantewang committed:
LPD-68357 Defer the creation of the OSGi servlet contexts' temp dir upon get

dantewang committed:
LPD-68357 We have very limited usages for this attribute and none are done with the OSGi HTTP servlet's context, so we can make it a rule for those who use it to "mkdirs" upon getting. No need to create here.

sandrodw3 committed:
LPD-68122 Migrate to ts

DavysonMelo committed:
LPD-65888 Change test to use object entry instead of message board

yuriymr committed:
LPD-66671 baseline

 # breaking

 ## What modules/apps/portal-workflow/portal-workflow-kaleo-definition-api/bnd.bnd

    Modifications on portal-workflow-kaleo-definition-api to remove Velocity Template from workflow

 ## Why

    Velocity is deprecated

----

JhossephG committed:
LPD-68036 feat: rename and create exception mapper

sandrodw3 committed:
LPD-68122 Support undo

JhossephG committed:
LPD-68037 test: add API coverage

sandrodw3 committed:
LPD-68122 Update panel when parent changes

sandrodw3 committed:
LPD-68122 Fix dnd validation for Form Relationship

CalebJamesH committed:
LPD-67547 Added SVGs

CalebJamesH committed:
LPD-67547 Add option to create new account on Product Purchase flow

CalebJamesH committed:
LPD-67547 SF

robertoDiaz committed:
LPD-68374 Add ObjectDefinitionSelectionFDSFilter for filtering in CMS

BryanEngler committed:
LPD-1490 portal-search: Rename method

https://liferay.atlassian.net/browse/LPD-1490

BryanEngler committed:
LPD-1490 portal-search: Contribute highlight fields when advanced search syntax is enabled

https://liferay.atlassian.net/browse/LPD-1490

BryanEngler committed:
LPD-1490 users-admin-impl: Extract highlight fields contribution to a HighlightFieldNamesQueryConfigContributor

https://liferay.atlassian.net/browse/LPD-1490

BryanEngler committed:
LPD-1490 calendar-service: Extract highlight fields contribution to a HighlightFieldNamesQueryConfigContributor

https://liferay.atlassian.net/browse/LPD-1490

BryanEngler committed:
LPD-1490 document-library-service: Extract highlight fields contribution to a HighlightFieldNamesQueryConfigContributor

https://liferay.atlassian.net/browse/LPD-1490

mariuo committed:
LPD-66189 Treat null singleRecipient

dsanz committed:
LPD-67390 Set PUSH as default behavior for config-in-URL

daiazius committed:
LPD-57780 Rename

chestofwonders committed:
LPD-68115 Fix behaviour issues detected on the dataset creation

dsanz committed:
LPD-67390 Not needed

dsanz committed:
LPD-67390 Turn config in URL feature off in the "Single selection" sample, to keep tests working

daiazius committed:
LPD-57780 Simplify

dsanz committed:
LPD-67390 Rename for better consistency

daiazius committed:
LPD-67917 Inline

manuelecastro committed:
LPD-68421 Ensure ServletResponse respects response buffers

BryanEngler committed:
LPD-1490 portal-search-spi: baseline

https://liferay.atlassian.net/browse/LPD-1490

BryanEngler committed:
LPD-1490 portal-search-spi: Add HighlightFieldNamesQueryConfigContributor interface

https://liferay.atlassian.net/browse/LPD-1490

BryanEngler committed:
LPD-1490 journal-service: Extract highlight fields contribution to a HighlightFieldNamesQueryConfigContributor

https://liferay.atlassian.net/browse/LPD-1490

DavysonMelo committed:
LPD-65888 Add wait for network idle to addComment function

victorg1991 committed:
LPD-68199 Take into account that we may be rendering a fragment dropzone that it may be inside a form

JhossephG committed:
LPD-68036 buildService

# breaking

## What modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectLayoutLocalServiceImpl.java

Rename exception class

## Why

Turn it flexible for both types: Categorization and SEO

----

bcarvalho89 committed:
LPD-68433 Add support for default permission modal in breadcrumb

BryanEngler committed:
LPD-1490 blogs-service: Extract highlight fields contribution to a HighlightFieldNamesQueryConfigContributor

https://liferay.atlassian.net/browse/LPD-1490

bcarvalho89 committed:
LPD-68433 Add default permission action in space breadcrumb

BryanEngler committed:
LPD-1490 journal-service: Extract highlight fields contribution to a HighlightFieldNamesQueryConfigContributor

https://liferay.atlassian.net/browse/LPD-1490

mariuo committed:
LPD-68239 Add unit test

BryanEngler committed:
LPD-1490 layout-impl: Extract highlight fields contribution to a HighlightFieldNamesQueryConfigContributor

https://liferay.atlassian.net/browse/LPD-1490

BryanEngler committed:
LPD-1490 message-boards-service: Extract highlight fields contribution to a HighlightFieldNamesQueryConfigContributor

https://liferay.atlassian.net/browse/LPD-1490

larissacribeiro committed:
LPD-68276 Sort

BryanEngler committed:
LPD-1490 object-service: Create generic contributor to cover ObjectEntry fields

https://liferay.atlassian.net/browse/LPD-1490

ctang3 committed:
LPD-66851 Derive the active component from the active state. Without this change, the `children` state is only set once during the initial render and the component wouldn't respond to the `children` prop changes.

ctang3 committed:
LPD-66851 Remove usages of CategorizationContentContainer and switch to using VerticalNavLayout instead

ctang3 committed:
LPD-66851 Add CSS improvement fix for correct form viewheight that CategorizationContentContainer had

ctang3 committed:
LPD-66851 Add data-testid for the vertical nav and rename EditCategoryPage's locator to the new data-testid name

ctang3 committed:
LPD-66851 Delete CategorizationContentContainer since usages have been switched to using VerticalNavLayout instead

thektan committed:
LPD-66625 Use the same labels as management toolbar taglib

dantewang committed:
LPD-63920 Clean up versions for Weblogic from app.server.properties, keep only 14.1.1 which will be the starting point to upgrade to 15.1.1.

dantewang committed:
LPD-63920 Clean up branching logic from build-dist.xml and build-test.xml, keep only the logic of 14.1.1.

adolfopa committed:
LPD-68433 Remove duplication

adolfopa committed:
LPD-68433 SF

adolfopa committed:
LPD-68433 Pass both class name and erc from the java code

sandrodw3 committed:
LPD-68199 Add dependency

CrescenzoRegaLF committed:
LPD-66996 Adds commerce payment method name terms to notification functionality

CrescenzoRegaLF committed:
LPD-66996 Adds payment method to Commerce Order System Object

CrescenzoRegaLF committed:
LPD-66996 Adds integration tests

CrescenzoRegaLF committed:
LPD-66996 Adds language key

CrescenzoRegaLF committed:
LPD-66996 Autogenerated Code (gw buildLang)

sandrodw3 committed:
LPD-68199 Export event

sandrodw3 committed:
LPD-68199 Update translation status when registering a localized input

sandrodw3 committed:
LPD-68199 Use state atom to store selected locale id

alc-camilla committed:
LRSD-8207 - Setup DXP Capabilities page without Product categories

LRSD-8207 - Setup DXP Capabilities page without Product categories

sandrodw3 committed:
LPD-68122 Add playwright test

sandrodw3 committed:
LPD-68122 Adapt tests

carolmariaabb committed:
LPD-68518 The value of defaultLanguageId must always be used to populate the insertedValues map

larissacribeiro committed:
LPD-68276 Add helper functions to clean up empty properties on page object

larissacribeiro committed:
LPD-68276 Replace function call

larissacribeiro committed:
LPD-68276 Add unit test

manuelecastro committed:
LPD-68421 Add JBoss to the MFA suite

carolmariaabb committed:
LPD-68518 Adjust the test to expose the issue that occurs when the default language ID is ordered after other locales that have data

alessiorendina committed:
LPD-68466 Added commerce checkout pre action to handle immediate checkout

alessiorendina committed:
LPD-68466 Removed unused components

alessiorendina committed:
LPD-68466 Hide account selector for guest users

alessiorendina committed:
LPD-68466 Moved creation buttons

alessiorendina committed:
LPD-68466 Moved all components to new folder

alessiorendina committed:
LPD-68466 Added commerce fragment utility class

alessiorendina committed:
LPD-68466 Added feature flag dependency

alessiorendina committed:
LPD-68466 Fixed site initializers

alessiorendina committed:
LPD-68466 Added new type definitions

alessiorendina committed:
LPD-68466 Refactor account selector taglib

alessiorendina committed:
LPD-68466 Removed immediate checkout handling from js

alessiorendina committed:
LPD-68466 Refactor fragment renderers

alessiorendina committed:
LPD-68466 Fixed css

alessiorendina committed:
LPD-68466 Added required js dependency

alessiorendina committed:
LPD-68466 Refactor display contexts

rafaprax committed:
LPD-57780 SF, buy space and method rename

alessiorendina committed:
LPD-68466 Rename

CalebJamesH committed:
LPD-67547 Refactor

kenjiheigel committed:
LRCI-6527 Add test suite for modules-compile

anabxalves committed:
LRSD-11375 - SF

thektan committed:
LPD-66619 Remove whitespace border in data set wrapper

jeffwu0724 committed:
LPD-68305 restore mailService after we finished using its proxy

jeffwu0724 committed:
LPD-68305 clean up left over sessions added by test

BryanEngler committed:
LPD-1490 portal-search-elasticsearch7-impl: Now that the highlight fields are contributed, we should require a field match

https://liferay.atlassian.net/browse/LPD-1490

daiazius committed:
LPD-67917 Sublime sort

gamerson committed:
LPD-67024 remove mechanism for overriding Configurator

gamerson committed:
LPD-67024 use latest org.apache.felix.configurator

gamerson committed:
LPD-67024 remove dead code

# breaking

## What portal-kernel/src/com/liferay/portal/kernel/zip/ZipFileUtil.java

Two public static methods that are not used anywhere were removed

## Why

These two methods were dead code and should be removed

----

gamerson committed:
LPD-67024 extract ZipFileTestUtil

fmarcio committed:
LPD-68548 update URL in BasePage regarding account settings

liferay-continuous-integration commented:
<html><h3>:x: ci:test:stable - 0 out of 1 jobs passed in 5 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/25cca0d75ec64ed6265f8101d9c9e9296e5bd620">25cca0d75ec64ed6265f8101d9c9e9296e5bd620</a></p><details><summary><strong>ci:test:stable - 0 out of 1 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-45.liferay.com/job/test-portal-acceptance-pullrequest(master)/487/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li></ul></div></details><h5>For more details click <a href="https://test-1-45.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/487/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-45.liferay.com/job/test-portal-acceptance-pullrequest(master)/487/">test-portal-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><div><p>The sender branch SHA could not be found on <strong><a href="https://github.com/kenjiheigel/liferay-portal/pull/1960">kenjiheigel/LRCI-6455-1</a></strong>. The sender branch may have been force pushed or deleted after the pull request test was initiated.</p><pre>fatal: Not a valid branch point: 'deb2fd4ce696bd7f48df7166acb8afa1a1ef4afe'.</pre></div></div></li></ol><h4>For upstream results, click <a href="https://test-1-41.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

BryanEngler committed:
LPD-1490 portal-search-opensearch2-impl: Now that the highlight fields are contributed, we should require a field match

https://liferay.atlassian.net/browse/LPD-1490

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-45.liferay.com/job/test-portal-acceptance-pullrequest(master)/487">test-portal-acceptance-pullrequest(master)#487</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-45.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/487/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>stable<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/kenjiheigel/liferay-portal/pull/1960">kenjiheigel#1960</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-45.liferay.com/job/publish-testray-report/209/">publish-testray-report#209</a><br/></div></div>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>stable</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-45.liferay.com/job/test-portal-acceptance-pullrequest(master)/490/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-45.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/490/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>stable</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-45.liferay.com/job/test-portal-acceptance-pullrequest(master)/491/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-45.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/491/jenkins-report.html">here</a>.</b></p>

jeffwu0724 committed:
LPD-68307 upgrade reactor-netty-core to 1.2.10 as it is needed by reactor-netty-http:1.2.10

jeffwu0724 committed:
LPD-68307 SF

michaelhashimoto committed:
LRCI-6429 Avoid unnecessary fields when getting testray case results from a build

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>stable</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-45.liferay.com/job/test-portal-acceptance-pullrequest(master)/493/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-45.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/493/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:stable - 1 out of 34 jobs passed in 37 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/25cca0d75ec64ed6265f8101d9c9e9296e5bd620">25cca0d75ec64ed6265f8101d9c9e9296e5bd620</a></p><details><summary><strong>ci:test:stable - 1 out of 34 jobs PASSED</strong></summary><div><h4>33 Failed Jobs:</h4><ul><li><a href="https://test-1-45.liferay.com/job/test-portal-acceptance-pullrequest(master)/492/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>build-iw<ul><li><a href="https://test-1-47.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/43631/">build-iw/0/0</a></li></ul></li><li>functional-upgrade-tomcat101-postgresql163_stable<ul><li><a href="https://test-1-46.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/51990/"><strike>functional-upgrade-tomcat101-postgresql163_stable/0/0</strike></a></li><li><a href="https://test-1-44.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/42784/"><strike>functional-upgrade-tomcat101-postgresql163_stable/0/1</strike></a></li><li><a href="https://test-1-47.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/43632/"><strike>functional-upgrade-tomcat101-postgresql163_stable/0/2</strike></a></li><li><a href="https://test-1-45.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/45290/"><strike>functional-upgrade-tomcat101-postgresql163_stable/0/3</strike></a></li><li><a href="https://test-1-43.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/44267/"><strike>functional-upgrade-tomcat101-postgresql163_stable/0/4</strike></a></li></ul></li><li>js-unit<ul><li><a href="https://test-1-42.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/72440/"><strike>js-unit/0/0</strike></a></li><li><a href="https://test-1-41.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/71726/"><strike>js-unit/1/0</strike></a></li><li><a href="https://test-1-42.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/72441/"><strike>js-unit/2/0</strike></a></li><li><a href="https://test-1-46.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/51991/"><strike>js-unit/3/0</strike></a></li></ul></li><li>modules-compile<ul><li><a href="https://test-1-47.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/43634/"><strike>modules-compile/0/0</strike></a></li><li><a href="https://test-1-43.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/44270/"><strike>modules-compile/0/1</strike></a></li><li><a href="https://test-1-45.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/45294/"><strike>modules-compile/0/2</strike></a></li><li><a href="https://test-1-43.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/44269/"><strike>modules-compile/0/3</strike></a></li><li><a href="https://test-1-42.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/72443/"><strike>modules-compile/0/4</strike></a></li><li><a href="https://test-1-45.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/45293/"><strike>modules-compile/0/5</strike></a></li><li><a href="https://test-1-42.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/72442/"><strike>modules-compile/0/6</strike></a></li><li><a href="https://test-1-48.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/44995/"><strike>modules-compile/0/7</strike></a></li><li><a href="https://test-1-46.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/51992/"><strike>modules-compile/0/8</strike></a></li><li><a href="https://test-1-48.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/44993/"><strike>modules-compile/0/9</strike></a></li><li><a href="https://test-1-42.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/72444/"><strike>modules-compile/0/10</strike></a></li><li><a href="https://test-1-44.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/42786/"><strike>modules-compile/0/11</strike></a></li><li><a href="https://test-1-45.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/45292/"><strike>modules-compile/0/12</strike></a></li></ul></li><li>modules-integration-postgresql163_stable<ul><li><a href="https://test-1-43.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/44271/"><strike>modules-integration-postgresql163_stable/0/0</strike></a></li><li><a href="https://test-1-48.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/44994/"><strike>modules-integration-postgresql163_stable/0/1</strike></a></li></ul></li><li>modules-unit_stable<ul><li><a href="https://test-1-48.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/44992/"><strike>modules-unit_stable/0/0</strike></a></li></ul></li><li>playwright-compile<ul><li><a href="https://test-1-41.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/71727/"><strike>playwright-compile/0/0</strike></a></li></ul></li><li>playwright-js-smoke-tomcat101-hypersonic20<ul><li><a href="https://test-1-45.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/45291/"><strike>playwright-js-smoke-tomcat101-hypersonic20/0/0</strike></a></li></ul></li><li>playwright-js-tomcat101-postgresql163_stable<ul><li><a href="https://test-1-48.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/44991/"><strike>playwright-js-tomcat101-postgresql163_stable/0/0</strike></a></li></ul></li><li>rest-builder<ul><li><a href="https://test-1-44.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/42783/"><strike>rest-builder/0/0</strike></a></li></ul></li><li>service-builder<ul><li><a href="https://test-1-43.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/44266/"><strike>service-builder/0/0</strike></a></li><li><a href="https://test-1-47.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/43630/"><strike>service-builder/0/1</strike></a></li></ul></li><li>unit_stable<ul><li><a href="https://test-1-42.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/72439/"><strike>unit_stable/0/0</strike></a></li></ul></li></ul></div><details><summary><strong>1 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-45.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/492/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-42.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/72440//consoleText">js-unit/0/0</a><pre>  at hudson.tasks.CommandInterpreter.perform(CommandInterpreter.java:92)
  at hudson.tasks.BuildStepMonitor\$1.perform(BuildStepMonitor.java:20)
  at hudson.model.AbstractBuild\$AbstractBuildExecution.perform(AbstractBuild.java:818)
  at hudson.model.Build\$BuildExecution.build(Build.java:199)
  at hudson.model.Build\$BuildExecution.doRun(Build.java:164)
  at hudson.model.AbstractBuild\$AbstractBuildExecution.run(AbstractBuild.java:526)
  at hudson.model.Run.execute(Run.java:1895)
  at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:44)
  at hudson.model.ResourceController.execute(ResourceController.java:101)
  at hudson.model.Executor.run(Executor.java:442)
Build step 'Execute shell' marked build as failure
[PostBuildScript] - [INFO] Executing post build scripts.
[PostBuildScript] - [INFO] Build does not have any of the results [ABORTED]. Did not execute build step #0.
ERROR: Step ‘Publish JUnit test result report’ failed: no workspace for test-portal-acceptance-pullrequest-downstream(master) #72440
[PostBuildScript] - [INFO] Executing post build scripts.
ERROR: Build step failed with exception
java.lang.NullPointerException: no such build node: i-01f53aade6f7ac21f
  at hudson.tasks.CommandInterpreter.perform(CommandInterpreter.java:112)
  at hudson.tasks.CommandInterpreter.perform(CommandInterpreter.java:92)
  at org.jenkinsci.plugins.postbuildscript.processor.Processor.processBuildSteps(Processor.java:180)
  at org.jenkinsci.plugins.postbuildscript.processor.Processor.processScripts(Processor.java:91)
  at org.jenkinsci.plugins.postbuildscript.processor.Processor.process(Processor.java:79)
  at org.jenkinsci.plugins.postbuildscript.processor.Processor.process(Processor.java:73)
  at org.jenkinsci.plugins.postbuildscript.PostBuildScript.perform(PostBuildScript.java:77)
  at hudson.tasks.BuildStepMonitor\$1.perform(BuildStepMonitor.java:20)
  at hudson.model.AbstractBuild\$AbstractBuildExecution.perform(AbstractBuild.java:818)
  at hudson.model.AbstractBuild\$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:767)
  at hudson.model.Build\$BuildExecution.post2(Build.java:179)
  at hudson.model.AbstractBuild\$AbstractBuildExecution.post(AbstractBuild.java:711)
  at hudson.model.Run.execute(Run.java:1918)
  at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:44)
  at hudson.model.ResourceController.execute(ResourceController.java:101)
  at hudson.model.Executor.run(Executor.java:442)
Build step 'Execute scripts' marked build as failure
Finished: FAILURE</pre></div></li><li><div><a href="https://test-1-42.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/72441//consoleText">js-unit/2/0</a><pre>  at hudson.tasks.CommandInterpreter.perform(CommandInterpreter.java:92)
  at hudson.tasks.BuildStepMonitor\$1.perform(BuildStepMonitor.java:20)
  at hudson.model.AbstractBuild\$AbstractBuildExecution.perform(AbstractBuild.java:818)
  at hudson.model.Build\$BuildExecution.build(Build.java:199)
  at hudson.model.Build\$BuildExecution.doRun(Build.java:164)
  at hudson.model.AbstractBuild\$AbstractBuildExecution.run(AbstractBuild.java:526)
  at hudson.model.Run.execute(Run.java:1895)
  at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:44)
  at hudson.model.ResourceController.execute(ResourceController.java:101)
  at hudson.model.Executor.run(Executor.java:442)
Build step 'Execute shell' marked build as failure
[PostBuildScript] - [INFO] Executing post build scripts.
[PostBuildScript] - [INFO] Build does not have any of the results [ABORTED]. Did not execute build step #0.
ERROR: Step ‘Publish JUnit test result report’ failed: no workspace for test-portal-acceptance-pullrequest-downstream(master) #72441
[PostBuildScript] - [INFO] Executing post build scripts.
ERROR: Build step failed with exception
java.lang.NullPointerException: no such build node: i-006b1fa244f2041c6
  at hudson.tasks.CommandInterpreter.perform(CommandInterpreter.java:112)
  at hudson.tasks.CommandInterpreter.perform(CommandInterpreter.java:92)
  at org.jenkinsci.plugins.postbuildscript.processor.Processor.processBuildSteps(Processor.java:180)
  at org.jenkinsci.plugins.postbuildscript.processor.Processor.processScripts(Processor.java:91)
  at org.jenkinsci.plugins.postbuildscript.processor.Processor.process(Processor.java:79)
  at org.jenkinsci.plugins.postbuildscript.processor.Processor.process(Processor.java:73)
  at org.jenkinsci.plugins.postbuildscript.PostBuildScript.perform(PostBuildScript.java:77)
  at hudson.tasks.BuildStepMonitor\$1.perform(BuildStepMonitor.java:20)
  at hudson.model.AbstractBuild\$AbstractBuildExecution.perform(AbstractBuild.java:818)
  at hudson.model.AbstractBuild\$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:767)
  at hudson.model.Build\$BuildExecution.post2(Build.java:179)
  at hudson.model.AbstractBuild\$AbstractBuildExecution.post(AbstractBuild.java:711)
  at hudson.model.Run.execute(Run.java:1918)
  at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:44)
  at hudson.model.ResourceController.execute(ResourceController.java:101)
  at hudson.model.Executor.run(Executor.java:442)
Build step 'Execute scripts' marked build as failure
Finished: FAILURE</pre></div></li><li><div><a href="https://test-1-46.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/51991//consoleText">js-unit/3/0</a><pre>  at hudson.tasks.CommandInterpreter.perform(CommandInterpreter.java:92)
  at hudson.tasks.BuildStepMonitor\$1.perform(BuildStepMonitor.java:20)
  at hudson.model.AbstractBuild\$AbstractBuildExecution.perform(AbstractBuild.java:818)
  at hudson.model.Build\$BuildExecution.build(Build.java:199)
  at hudson.model.Build\$BuildExecution.doRun(Build.java:164)
  at hudson.model.AbstractBuild\$AbstractBuildExecution.run(AbstractBuild.java:526)
  at hudson.model.Run.execute(Run.java:1895)
  at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:44)
  at hudson.model.ResourceController.execute(ResourceController.java:101)
  at hudson.model.Executor.run(Executor.java:442)
Build step 'Execute shell' marked build as failure
[PostBuildScript] - [INFO] Executing post build scripts.
[PostBuildScript] - [INFO] Build does not have any of the results [ABORTED]. Did not execute build step #0.
ERROR: Step ‘Publish JUnit test result report’ failed: no workspace for test-portal-acceptance-pullrequest-downstream(master) #51991
[PostBuildScript] - [INFO] Executing post build scripts.
ERROR: Build step failed with exception
java.lang.NullPointerException: no such build node: i-0ba54078e4ec799ec
  at hudson.tasks.CommandInterpreter.perform(CommandInterpreter.java:112)
  at hudson.tasks.CommandInterpreter.perform(CommandInterpreter.java:92)
  at org.jenkinsci.plugins.postbuildscript.processor.Processor.processBuildSteps(Processor.java:180)
  at org.jenkinsci.plugins.postbuildscript.processor.Processor.processScripts(Processor.java:91)
  at org.jenkinsci.plugins.postbuildscript.processor.Processor.process(Processor.java:79)
  at org.jenkinsci.plugins.postbuildscript.processor.Processor.process(Processor.java:73)
  at org.jenkinsci.plugins.postbuildscript.PostBuildScript.perform(PostBuildScript.java:77)
  at hudson.tasks.BuildStepMonitor\$1.perform(BuildStepMonitor.java:20)
  at hudson.model.AbstractBuild\$AbstractBuildExecution.perform(AbstractBuild.java:818)
  at hudson.model.AbstractBuild\$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:767)
  at hudson.model.Build\$BuildExecution.post2(Build.java:179)
  at hudson.model.AbstractBuild\$AbstractBuildExecution.post(AbstractBuild.java:711)
  at hudson.model.Run.execute(Run.java:1918)
  at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:44)
  at hudson.model.ResourceController.execute(ResourceController.java:101)
  at hudson.model.Executor.run(Executor.java:442)
Build step 'Execute scripts' marked build as failure
Finished: FAILURE</pre></div></li><li><div><a href="https://test-1-46.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/51992//consoleText">modules-compile/0/8</a><pre>  at hudson.tasks.CommandInterpreter.perform(CommandInterpreter.java:92)
  at hudson.tasks.BuildStepMonitor\$1.perform(BuildStepMonitor.java:20)
  at hudson.model.AbstractBuild\$AbstractBuildExecution.perform(AbstractBuild.java:818)
  at hudson.model.Build\$BuildExecution.build(Build.java:199)
  at hudson.model.Build\$BuildExecution.doRun(Build.java:164)
  at hudson.model.AbstractBuild\$AbstractBuildExecution.run(AbstractBuild.java:526)
  at hudson.model.Run.execute(Run.java:1895)
  at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:44)
  at hudson.model.ResourceController.execute(ResourceController.java:101)
  at hudson.model.Executor.run(Executor.java:442)
Build step 'Execute shell' marked build as failure
[PostBuildScript] - [INFO] Executing post build scripts.
[PostBuildScript] - [INFO] Build does not have any of the results [ABORTED]. Did not execute build step #0.
ERROR: Step ‘Publish JUnit test result report’ failed: no workspace for test-portal-acceptance-pullrequest-downstream(master) #51992
[PostBuildScript] - [INFO] Executing post build scripts.
ERROR: Build step failed with exception
java.lang.NullPointerException: no such build node: i-0bc19f356016d82b9
  at hudson.tasks.CommandInterpreter.perform(CommandInterpreter.java:112)
  at hudson.tasks.CommandInterpreter.perform(CommandInterpreter.java:92)
  at org.jenkinsci.plugins.postbuildscript.processor.Processor.processBuildSteps(Processor.java:180)
  at org.jenkinsci.plugins.postbuildscript.processor.Processor.processScripts(Processor.java:91)
  at org.jenkinsci.plugins.postbuildscript.processor.Processor.process(Processor.java:79)
  at org.jenkinsci.plugins.postbuildscript.processor.Processor.process(Processor.java:73)
  at org.jenkinsci.plugins.postbuildscript.PostBuildScript.perform(PostBuildScript.java:77)
  at hudson.tasks.BuildStepMonitor\$1.perform(BuildStepMonitor.java:20)
  at hudson.model.AbstractBuild\$AbstractBuildExecution.perform(AbstractBuild.java:818)
  at hudson.model.AbstractBuild\$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:767)
  at hudson.model.Build\$BuildExecution.post2(Build.java:179)
  at hudson.model.AbstractBuild\$AbstractBuildExecution.post(AbstractBuild.java:711)
  at hudson.model.Run.execute(Run.java:1918)
  at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:44)
  at hudson.model.ResourceController.execute(ResourceController.java:101)
  at hudson.model.Executor.run(Executor.java:442)
Build step 'Execute scripts' marked build as failure
Finished: FAILURE</pre></div></li><li><div><a href="https://test-1-43.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/44271//consoleText">modules-integration-postgresql163_stable/0/0</a><pre>  at hudson.tasks.CommandInterpreter.perform(CommandInterpreter.java:92)
  at hudson.tasks.BuildStepMonitor\$1.perform(BuildStepMonitor.java:20)
  at hudson.model.AbstractBuild\$AbstractBuildExecution.perform(AbstractBuild.java:818)
  at hudson.model.Build\$BuildExecution.build(Build.java:199)
  at hudson.model.Build\$BuildExecution.doRun(Build.java:164)
  at hudson.model.AbstractBuild\$AbstractBuildExecution.run(AbstractBuild.java:526)
  at hudson.model.Run.execute(Run.java:1895)
  at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:44)
  at hudson.model.ResourceController.execute(ResourceController.java:101)
  at hudson.model.Executor.run(Executor.java:442)
Build step 'Execute shell' marked build as failure
[PostBuildScript] - [INFO] Executing post build scripts.
[PostBuildScript] - [INFO] Build does not have any of the results [ABORTED]. Did not execute build step #0.
ERROR: Step ‘Publish JUnit test result report’ failed: no workspace for test-portal-acceptance-pullrequest-downstream(master) #44271
[PostBuildScript] - [INFO] Executing post build scripts.
ERROR: Build step failed with exception
java.lang.NullPointerException: no such build node: i-0b568c47fa0e03664
  at hudson.tasks.CommandInterpreter.perform(CommandInterpreter.java:112)
  at hudson.tasks.CommandInterpreter.perform(CommandInterpreter.java:92)
  at org.jenkinsci.plugins.postbuildscript.processor.Processor.processBuildSteps(Processor.java:180)
  at org.jenkinsci.plugins.postbuildscript.processor.Processor.processScripts(Processor.java:91)
  at org.jenkinsci.plugins.postbuildscript.processor.Processor.process(Processor.java:79)
  at org.jenkinsci.plugins.postbuildscript.processor.Processor.process(Processor.java:73)
  at org.jenkinsci.plugins.postbuildscript.PostBuildScript.perform(PostBuildScript.java:77)
  at hudson.tasks.BuildStepMonitor\$1.perform(BuildStepMonitor.java:20)
  at hudson.model.AbstractBuild\$AbstractBuildExecution.perform(AbstractBuild.java:818)
  at hudson.model.AbstractBuild\$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:767)
  at hudson.model.Build\$BuildExecution.post2(Build.java:179)
  at hudson.model.AbstractBuild\$AbstractBuildExecution.post(AbstractBuild.java:711)
  at hudson.model.Run.execute(Run.java:1918)
  at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:44)
  at hudson.model.ResourceController.execute(ResourceController.java:101)
  at hudson.model.Executor.run(Executor.java:442)
Build step 'Execute scripts' marked build as failure
Finished: FAILURE</pre></div></li><li>...</li></ol><h4>For upstream results, click <a href="https://test-1-41.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

jeffwu0724 committed:
LPD-68307 auto generate, by running "ant build-lib-versions"

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-45.liferay.com/job/test-portal-acceptance-pullrequest(master)/492">test-portal-acceptance-pullrequest(master)#492</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-45.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/492/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>stable<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/kenjiheigel/liferay-portal/pull/1960">kenjiheigel#1960</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/339738520">[master] ci:test:stable - kenjiheigel &gt; kenjiheigel - PR#1960 - 2025-10-14[16:16:08]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>339738520</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-45.liferay.com/job/test-portal-acceptance-pullrequest(master)/492/">test-portal-acceptance-pullrequest(master)#492</a><br/></div></div>

alessiorendina committed:
LPD-68466 Regen services

markocikos committed:
LPD-67267 yarn run format

victorg1991 committed:
LPD-68618 Perform different approach since doAsUserId implementation has changed

victorg1991 committed:
LPD-68618 Remove parameter

albertojml committed:
LPD-68532 Calculate the property type properly

alessiorendina committed:
LPD-68466 Semantic versioning (gw baseline)

alessiorendina committed:
LPD-68466 Fixed account selector unit tests

alessiorendina committed:
LPD-68466 Fixed playwright tests

marcogalluzzi committed:
LPD-68012 Partially revert changes done in LPD-63562 and LPD-65787

bcarvalho89 committed:
LPD-68613 Rename component to StatusLabel

bcarvalho89 committed:
LPD-68613 Standardize status label text and type

bcarvalho89 committed:
LPD-68613 Fix missing translation text call

bcarvalho89 committed:
LPD-68613 SF

bcarvalho89 committed:
LPD-68613 Use JSX syntax instead of function

albertojml committed:
LPD-68532 Add endpoint for testing previously failing usecase

alessiorendina committed:
LPD-68466 Fixed potential security risk

albertojml committed:
LPD-68532 BuildREST in test module

carolmariaabb committed:
LPD-68036 Refactor test

marcogalluzzi committed:
LPD-68012 auto baseline

4lejandrito committed:
LPD-66480 Fixing 415 errors after refactor

nicolas-sss committed:
LPD-30966 Fix message line number

victorg1991 committed:
LPD-68645 Don't show repeatable buttons in view mode

victorg1991 committed:
LPD-68645 Update test to check view mode

adolfopa committed:
LPD-68012 Auto SF

BryanEngler committed:
LPD-1490 portal-web: Activate the test and update its description

https://liferay.atlassian.net/browse/LPD-1490

monaco-fabio committed:
LPD-61508 Fix closing dropdown on infoPanel

monaco-fabio committed:
LPD-61508 Fix css on Info Panel

ivicac committed:
LPD-61901 Fix the quantity field displayed in scientific notation

brittneyq committed:
LRCI-6423 Change mysql57 references to mysql84 in test.properties files

brbalazs committed:
LPD-67582 Unit test

brbalazs committed:
LPD-67582 Playwright test

brbalazs committed:
LPD-67582 Auto-generated (buildLang)

gamerson committed:
LPD-67024 refactor InMemoryThreadLocal to use SafeCloseable API

# breaking

## What modules/apps/static/portal-configuration/portal-configuration-persistence-api/src/main/java/com/liferay/portal/configuration/persistence/InMemoryOnlyConfigurationThreadLocal.java

Remove set() method and replaced with  setWithSafeClosable() version

## Why

The previous set() method did not handle the tri-state nature of thread
local booleans that they can be true/false/unset

----

gamerson committed:
LPD-67024 add new bundle tracker to manage embedded client extension configuration

ricardomariz committed:
LPD-63834 Add safe optional chaining

ericyanLr committed:
LPD-58634 Fix test; order for the expected arrays changed after sort/rename commits (6f1b7598 + 1625c7bf)

ericyanLr committed:
LPD-58634 SF, sort parameters

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 1 out of 1 jobs passed in 22 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/25cca0d75ec64ed6265f8101d9c9e9296e5bd620">25cca0d75ec64ed6265f8101d9c9e9296e5bd620</a></p><details><summary><strong>ci:test:stable - 1 out of 1 jobs PASSED</strong></summary></details><h5>For more details click <a href="https://test-1-45.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/495/jenkins-report.html">here</a>.</h5></details></html>

JhossephG committed:
LPD-68036 enable setting to test

kenjiheigel committed:
LRCI-6537 Always update JenkinsMaster json objects when getting slave in AWS

ambrinchaudhary committed:
LPD-65093 Hide cancel button if there is no url for it (CMS doesn't have it as it is rendered in a modal)

ambrinchaudhary committed:
LPD-65093 Also hide Done button when there is nowhere to redirect (in CMS it is open in a modal)

brianchandotcom committed:
LPD-57780 SF

brianchandotcom committed:
LPD-68374 SF

pedroliveirav committed:
LRSD-11335 Update patching and release category

michaelhashimoto committed:
LRCI-6429 Optimized determining the testray attachments by returning less fields

michaelhashimoto committed:
LRCI-6429 Determing the overhead durations by removing the test execution duration if it is available

michaelhashimoto committed:
LRCI-6429 Rename

ryanschuhler committed:
LRSD-11397 Ticket attachment is uploaded even after user clicks "Cancel Upload"

michaelhashimoto committed:
LRCI-6429 Reduce default test durations for modules integration & unit tests

ryanschuhler committed:
LRSD-11337 Update Attachment links to point to new ticket-attachments route

brianchandotcom committed:
LRCI-6429 SF

brianchandotcom committed:
LRCI-6429 prep next

ling-alan-huang committed:
LPD-63694 SF, remove unnecessary semicolon in runSQL

ling-alan-huang committed:
LPD-63694 SF, remove unnecessary semicolon in runSQL

antonio-ortega committed:
LPD-68563 This filter should only apply to JS files, otherwise we might damage files like images

sandrodw3 committed:
LPD-68618 Add missing await

brbalazs committed:
LPD-67582 Integration test updated

liferay-translation committed:
LPS-77699 Update Translations

brianchandotcom committed:
LPD-67523 SF

brianchandotcom committed:
LPD-68036 SF

Mylena-Rodrigues committed:
LRSD-8207 SF

carolmariaabb committed:
LPD-68449 Remove the `[%OBJECT_ENTRY_ASSIGNEE%]` term that will be replaced by the object field term

carolmariaabb committed:
LPD-68449 Move the logic to evaluate the assignee object field term

carolmariaabb committed:
LPD-68449 Adjust integration tests

Commit:
LPD-68013 - Reorder "Shared With Me" menu item.

brianchandotcom committed:
LPD-68532 prep next

gamerson committed:
LPD-67024 semver

rafaellajordao committed:
LPD-59848 Add aria-label tags to all drop-down menus with search fields in the CMS Dashboard

brianchandotcom committed:
LPD-30966 prep next

brianchandotcom committed:
LPD-30966 LPD-68532 prep next

brianchandotcom committed:
LPD-58634 Sort usage too

Commit:
LPD-68013 - Update numbering based on new order.

brianchandotcom committed:
LRSD-8207 SF

brianchandotcom committed:
LPD-68207 Test-fix

kenjiheigel committed:
LRCI-6539 Only send notifications on the final failure

kenjiheigel committed:
LRCI-6541 Add property to enable/disable checksum validation; we'll rely on AWS's automatic checksum validation for now

bcarvalho89 committed:
LPD-68613 Component inside a tableCell render should be a function instead of JSX

CalumR23 committed:
LRCI-6543 Remove loaded projects names

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-45.liferay.com/job/test-portal-acceptance-pullrequest(master)/496">test-portal-acceptance-pullrequest(master)#496</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-45.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/496/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>search<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/kenjiheigel/liferay-portal/pull/1960">kenjiheigel#1960</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/341224140/routines/341271546">[master] ci:test:search - kenjiheigel &gt; kenjiheigel - PR#1960 - 2025-10-15[14:27:53] # 2</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/341224140/routines/341271546/build/341271628">[master] ci:test:stable - 3741 - 2025-10-15[08:33:48]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>341271628</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-45.liferay.com/job/publish-testray-report/212/">publish-testray-report#212</a><br/></div></div>

brianchandotcom committed:
LPD-67024 SF

brianchandotcom committed:
LPD-67024 prep next

ctang3 committed:
LPD-53238 Add missing language key for the ADD_REMOTE resource action

ctang3 committed:
LPD-53238 buildLang

ling-alan-huang committed:
LRSD-11375 Auto SF

dannielraposo committed:
LPD-68532 Fix the template

dannielraposo committed:
LPD-68532 buildREST

liferay-translation committed:
LPS-77699 Update Translations

brianchandotcom committed:
LRCI-6539 Not needed

brianchandotcom committed:
LRCI-6539 prep next

brianchandotcom committed:
LPD-68532 prep next

brianchandotcom committed:
LPD-68532 prep next
